test/lib/httpReceiver.test.js

```diff
@@ -391,7 +392,7 @@ describe('http receiver', () => {
   it('should send a response', (done) => {
     function taskReceivedCallback(receivedTask, callback) {
       receivedTask.should.be.an.Object();
-      receivedTask.taskType.should.eql('dataLink');
+      receivedTask.taskType.should.eql('data');
```

Comment:
Yes.  The flex-sdk accepts both.  The httpReceiver makes the decision based on other things.  dataLink / businessLogic in mapi/kcs taskType strings still apply.